scanctl/cli.py

```diff
@@ -18,6 +18,8 @@
 import asyncio
 import logging
 import re
+import os
+import os.path
```

Comment:
Why import both? You can make do with just `os`.

Also, we should add a linter check (`flake8`) and potentially even a formatter (`black`); Imports should be alphabetical.

Reply:
Not sure why I thought it was required to import `os.path` separately from `os`. 

There is a linter check: https://github.com/spotify/scanctl/blob/73b98c65223556acbd7ad0ae5eeb257d4ee8c0ae/tox.ini#L4-L6

Feel free to add `black`. I tried it locally and didn't like the results, so I'll leave it to you.